airflow/providers/cncf/kubernetes/utils/pod_launcher.py

```diff
@@ -183,7 +186,12 @@ def parse_log_line(self, line: str) -> Tuple[str, str]:
             raise Exception(f'Log not in "{{timestamp}} {{log}}" format. Got: {line}')
         timestamp = line[:split_at]
         message = line[split_at + 1 :].rstrip()
-        return timestamp, message
+        try:
+            last_log_time = pendulum.parse(timestamp)
+        except ParserError:
+            self.log.error("Error parsing timestamp. Will continue execution but won't update timestamp")
```

Comment:
Print the line

Reply:
@ashb using a print statement?

Reply:
https://github.com/apache/airflow/pull/15638/files#diff-aabaf0e6b4525941e99892b17f18839d00dcaab0432c8a0828910a4dd4d9e003R149 @ashb the line gets printed here